docs/source/overview.rst

```diff
@@ -0,0 +1,20 @@
+Overview of Iroha
+=================
+
+What are the key features of Iroha?
+-----------------------------------
+
+Where Iroha can be used?
+------------------------
+
+How it is different from Bitcoin or Ethereum?
+---------------------------------------------
+
+How it is different from the rest of Hyperledger frameworks?
+------------------------------------------------------------
+
```

Comment:
Also comparison with some other permissoned blockchain platforms can be added: like Tendermint, Chain, etc.

Reply:
Paraphrased it